mail-filter/mimedefang/mimedefang-2.84.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils user
+
+DESCRIPTION="Antispam, antivirus and other customizable filters for MTAs with Milter support"
+HOMEPAGE="http://www.mimedefang.org/"
+SRC_URI="http://www.mimedefang.org/static/${P}.tar.gz"
+LICENSE="GPL-2+"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="clamav +poll"
+
+DEPEND=">=dev-perl/MIME-tools-5.412
+	dev-perl/IO-stringy
+	virtual/perl-MIME-Base64
+	dev-perl/Digest-SHA1
+	dev-perl/MailTools
+	dev-perl/Unix-Syslog
+	clamav? ( app-antivirus/clamav )
+	|| ( mail-filter/libmilter mail-mta/sendmail )"
+RDEPEND="${DEPEND}"
+RESTRICT="test"
+
+pkg_setup() {
+	enewgroup defang
+	enewuser defang -1 -1 -1 defang
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-2.72-ldflags.patch"
+	eapply_user
+}
+
+src_configure() {
+	econf \
+		--with-user=defang \
+		$(use_enable poll) \
+		$(use_enable clamav) \
+		$(use_enable clamav clamd)
+}
+
+src_install() {
+	emake DESTDIR="${D}" INSTALL_STRIP_FLAG="" install
+
+	fowners defang:defang /etc/mail/mimedefang-filter
+	fperms 644 /etc/mail/mimedefang-filter
+	insinto /etc/mail/
+	insopts -m 644
```

Comment:
I'm pretty sure that's the default mode.

Reply:
l-Term-ANSIColor-4.40.0-r1:0/0::ge

Reply:
Please see new commit.